src/core/type1_parser.js

```diff
@@ -150,6 +150,10 @@ var Type1CharString = (function Type1CharStringClosure() {
                 break;
               }
               subrNumber = this.stack.pop();
+              if (!subrs[subrNumber]) {
```

Comment:
This isn't necessary to fix the broken PDF, but it something I noticed when we were previously trying to decrypt the charstring.